vscode/src/createProject.ts

```diff
@@ -123,7 +123,7 @@ export async function initProjectCreator(context: vscode.ExtensionContext) {
           return;
         }
 
-        async function getQsFilesInDir(dir: vscode.Uri) {
+        async function getQsAndQscFilesInDir(dir: vscode.Uri) {
```

Comment:
Suggestion: call this `getSourceFilesInDir` to avoid the tiresome "qs and qsc" repetition

Reply:
done